core/src/builder.ts

```diff
@@ -77,13 +86,27 @@ function PropertyBuilder<Model>(initialProps?: Partial<Model>) {
 function Builder<Model>({
   generator,
   transformers,
+  type,
+  postBuild,
```

Comment:
In the new pattern, this is a callback used to manipulate the output of the build before it's returned to the consumer. This allows to resolve dependent properties of the data model.